### DISCUSSION TITLE
When deployed on Vercel, my swagger ui shows a blank page

### DISCUSSION THREAD

**Discussion by EdisonMatos** [vercel/vercel#10558] [Help]
### Question

Hello, guys. 

Im having the follow issue, on localhost it works fine, and, on AWS EC2 server too, but when i deply the application on Vercel, the path that should show my Swagger UI (like: localhost:8080/swagger), it only shows a blank page. As i said, when i deploy anywherelse it works fine, with the exactly same repositoy, but on vercel, it show a blank page.

Someone have any ideia about what can be happening? I've seached about it but without success.

Ps: I realized that some people have a issue that the swagger ui load but doesnt load the css files, but that is not my case. In my case i have a literally blank screen.

Thank you very much.

My swagger version: 7.0.3
My node version: 18.17.0

**Comment by hasbrak** [comment]
Hello EdisonMatos, 

I`ve the same issue, did you resolved it ?

Thanks in advance
